packages/forms/src/UIForm/Message/generateId.js

```diff
@@ -0,0 +1,11 @@
+export function generateDescriptionId(id) {
+	return id && `${id}-description`;
+}
+
+export function generateErrorId(id) {
+	return id && `${id}-error`;
+}
+
+export function generateDescribedBy(id) {
+	return id && `${generateDescriptionId(id)} ${generateErrorId(id)}`;
+}
```

Comment:
Would be better if the method returned something meaningful if the id isn't passed.
In that case it returns `undefined` if no `id` is passed.

Reply:
I changed that to generate an id via uuid.v4() if there is no base id.
So each widget now generate it, and pass it to the input and FieldTemplate to set it on description and errors